libr/bin/format/elf/elf.c

```diff
@@ -3502,14 +3519,20 @@ static RBinElfSymbol* Elf_(_r_bin_elf_get_symbols_imports)(ELFOBJ *bin, int type
 			memset (ret + ret_size, 0, nsym * sizeof (RBinElfSymbol));
 			prev_ret_size = ret_size;
 			ret_size += nsym;
+			symbols_by_name = ht_pp_new0 ();
+			for (k = 0; k < prev_ret_size; k++) {
```

Comment:
doesn't this loop re-evaluate elements that were already evaluated in the previous iteration?

Reply:
You mean previous iteration of section loop? Yes, but `ret` array gets relocated which invalidates the old pointers. It shouldn't be a problem unless there is large amount of symbol sections, can there be more than 2?